src/persist-client/src/lib.rs

```diff
@@ -283,6 +286,7 @@ impl PersistConfig {
             compaction_memory_bound_bytes: 1024 * MB,
             compaction_heuristic_min_inputs: 8,
             compaction_heuristic_min_updates: 1024,
+            compaction_reads_max_outstanding_parts: 8,
```

Comment:
chosen by fair dice roll? :sweat_smile:

Reply:
Roughly. Was aiming to match `compaction_heuristic_min_inputs` so for a request w/ the minimum number of batches they can all be fetched in parallel. And given S3 bandwidth, 8 parallel reads is somewhere in the range of a lot, but short of hogging all the network bandwidth for the pod

Reply:
can we stick this comment in the "tuning notes" comment in this file please